pom.xml

```diff
@@ -116,7 +116,7 @@
 		<dependency>
 			<groupId>com.nimbusds</groupId>
 			<artifactId>nimbus-jose-jwt</artifactId>
-			<version>7.5.1</version>
+			<version>7.9</version>
 		</dependency>
 		
```

Comment:
We can delete this pull request now since we don't use it anymore.

Reply:
OK, will do that.